src/main/java/seedu/address/logic/parser/EditMeetingCommandParser.java

```diff
@@ -0,0 +1,98 @@
+import seedu.address.logic.commands.EditMeetingCommand;
+import seedu.address.logic.commands.EditMeetingCommand.EditMeetingDescriptor;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.tag.Tag;
+
+public class EditMeetingCommandParser implements Parser<EditMeetingCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the EditMeetingCommand
+     * and returns an EditMeetingCommand object for execution.
+     *
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    public EditMeetingCommand parse(String args) throws ParseException {
+        requireNonNull(args);
+        ArgumentMultimap argMultiMap =
+                ArgumentTokenizer.tokenize(args, PREFIX_MEETING_NAME, PREFIX_DATE, PREFIX_START_TIME, PREFIX_END_TIME,
+                        PREFIX_PARTICIPANTS, PREFIX_TAG);
+        Index index;
+
+        try {
+            index = ParserUtil.parseIndex(argMultiMap.getPreamble());
+        } catch (ParseException pe) {
+            throw new
+                    ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, EditMeetingCommand.MESSAGE_USAGE), pe);
+        }
+
+        EditMeetingDescriptor editMeetingDescriptor = new EditMeetingDescriptor();
+        if (argMultiMap.getValue(PREFIX_MEETING_NAME).isPresent()) {
+            editMeetingDescriptor
+                    .setMeetingName(ParserUtil.parseMeetingName(argMultiMap.getValue(PREFIX_MEETING_NAME).get()));
+        }
+        if (argMultiMap.getValue(PREFIX_DATE).isPresent()) {
+            editMeetingDescriptor
+                    .setMeetingDate(ParserUtil.parseMeetingDate(argMultiMap.getValue(PREFIX_DATE).get()));
+        }
+        if (argMultiMap.getValue(PREFIX_START_TIME).isPresent()) {
+            editMeetingDescriptor
+                    .setStartTime(ParserUtil.parseStartTime(argMultiMap.getValue(PREFIX_START_TIME).get()));
+        }
+        if (argMultiMap.getValue(PREFIX_END_TIME).isPresent()) {
+            editMeetingDescriptor
+                    .setEndTime(ParserUtil.parseEndTime(argMultiMap.getValue(PREFIX_END_TIME).get()));
+        }
+        parseTagsForEdit(argMultiMap.getAllValues(PREFIX_TAG)).ifPresent(editMeetingDescriptor::setTags);
+        parseParticipantsIndexForEdit(argMultiMap.getAllValues(PREFIX_PARTICIPANTS))
+                .ifPresent((editMeetingDescriptor::setParticipantsIndex));
+
+        if (!editMeetingDescriptor.isAnyFieldEdited()) {
+            throw new ParseException(EditContactCommand.MESSAGE_NOT_EDITED);
+        }
+        return new EditMeetingCommand(index, editMeetingDescriptor);
+    }
+
+    /**
+     * Parses {@code Collection<String> tags} into a {@code Set<Tag>} if {@code tags} is non-empty.
+     * If {@code tags} contain only one element which is an empty string, it will be parsed into a
+     * {@code Set<Tag>} containing zero tags.
+     */
```

Comment:
Could possibly just do a similar write up for parseParticipantsIndexForEdit as well!